docs/tutorials/plots.rst

```diff
@@ -406,6 +406,43 @@ The default line for time-dependent plots is solid and the default label
 See the `Matplotlib`_ documentation for information on plotting styles in line
 plots.
 
+Night vision mode
```

Comment:
I slightly prefer "dark" and "light" style.
So my suggestion would be "Dark style" instead of "Night vision mode" here.
But @bmorris3, feel free to keep as is if you prefer it.